docs/middleware/ics29-fee/fee-distribution.md

```diff
@@ -0,0 +1,90 @@
+<!--
+order: 4
+-->
+
+# Fee Distribution
+
+Learn about payee registration for distribution of packet fees. The following document is intended for relayer operators {synopsis}
+
+## Pre-requisite Readings
+
+* [Fee Middleware](overview.md) {prereq}
+
+Packet fees are divided into 3 distinct amounts in order to compensate relayer operators for packet relaying on fee enabled IBC channels.
+
+- `RecvFee`: The sum of all packet receive fees distributed to a payee for successful execution of `MsgRecvPacket`.
+- `AckFee`: The sum of all packet acknowledgement fees distributed to a payee for successful execution of `MsgAcknowledgement`.
+- `TimeoutFee`: The sum of all packet timeout fees distributed to a payee for successful execution of `MsgTimeout`.
+
+## Register a payee address for forward relaying
+
+As mentioned previously in [ICS29 Concepts](../ics29-fee/overview.md#concepts), the forward relayer describes the actor who performs the submission of `MsgRecvPacket` on the destination chain. 
```

Comment:
I think I would remove "previously" since nobody guarantees that the reader will have read the other section before...

```suggestion
As mentioned in [ICS29 Concepts](../ics29-fee/overview.md#concepts), the forward relayer describes the actor who performs the submission of `MsgRecvPacket` on the destination chain. 
```